Source/WebCore/platform/video-codecs/cocoa/RTCVideoDecoderVTBAV1.h

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright (C) 2022 Apple Inc. All rights reserved.
```

Comment:
2023?

Reply:
I guess we could do 2022-2023.
I kept 2022 here.